component/navbar/navbar.html

```diff
@@ -144,6 +145,64 @@ <h1>Navbar</h1>
+                            <p class="hamburger-items">item 3</p>
+                            <p class="hamburger-items">item 4</p>
+                        </section>
+                    </div>
+                    <p class="navbar-title">R <span class="nav-title-short">& </span>B <span class="nav-title-short">UI </span> </p>
+                    
+                    <div class="navbar-utilities">
+                        <input type="text" class="searchbar" placeholder="Search Here">
+                        <i class="fas fa-heart navbar-icons"></i>
+                        <i class="fas fa-shopping-cart navbar-icons"></i>
+                        <i class="fas fa-user-circle navbar-icons"></i>
+                    </div>
+                </div>
+        </div>
+        <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%280%2C0%2C0%2C1%29&t=base16-dark&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=55px&wc=true&wa=true&pv=40px&ph=48px&ln=false&fl=1&fm=Hack&fs=13.5px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522navbar-component%2522%253E%250A%253Cdiv%2520class%253D%2522hamburger%2522%253E%250A%253Cdiv%2520class%253D%2522hamburger-lines%2522%253E%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522hamburger-lines%2522%253E%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522hamburger-lines%2522%253E%253C%252Fdiv%253E%250A%2520%253Csection%2520class%253D%2522navbar-hamburger-submenu%2522%253E%250A%2520%2520%253Cp%2520class%253D%2522hamburger-items%2522%253Eitem%25201%253C%252Fp%253E%250A%2520%2520%253Cp%2520class%253D%2522hamburger-items%2522%253Eitem%25202%253C%252Fp%253E%250A%2520%2520%253Cp%2520class%253D%2522hamburger-items%2522%253Eitem%25203%253C%252Fp%253E%250A%2520%2520%253Cp%2520class%253D%2522hamburger-items%2522%253Eitem%25204%253C%252Fp%253E%250A%2520%253C%252Fsection%253E%250A%253C%252Fdiv%253E%250A%253Cp%2520class%253D%2522navbar-title%2522%253ER%2520%253Cspan%2520class%253D%2522nav-title-short%2522%253E%2526%2520%253C%252Fspan%253EB%2520%253Cspan%2520class%253D%2522nav-title-short%2522%253EUI%2520%253C%252Fspan%253E%2520%253C%252Fp%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%250A%2520%253Cdiv%2520class%253D%2522navbar-utilities%2522%253E%250A%2520%2520%253Cinput%2520type%253D%2522text%2522%2520class%253D%2522searchbar%2522%2520placeholder%253D%2522Search%2520Here%2522%253E%250A%2520%2520%253Ci%2520class%253D%2522fas%2520fa-heart%2520navbar-icons%2522%253E%253C%252Fi%253E%250A%2520%2520%253Ci%2520class%253D%2522fas%2520fa-shopping-cart%2520navbar-icons%2522%253E%253C%252Fi%253E%250A%2520%2520%253Ci%2520class%253D%2522fas%2520fa-user-circle%2520navbar-icons%2522%253E%253C%252Fi%253E%250A%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520"
+  style="width: 1024px; height: 550px; border:0; transform: scale(1); overflow:hidden;"
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
```

Comment:
try to align code  properly, it will give good readability. Rest everything looks good